wireless_modules/middle_module/boot.py

```diff
@@ -0,0 +1,58 @@
+# Author: Kunj
+# Last updated: 1/08/2020
+
+# Import relevant libraries
+import esp
+import network # To connect to the WIFI
+import config # File in the same path containing WIFI credentials and Pin to in built blue LED
+import time
+
+
+def do_connect(essid, password):
+    """
+    Tries to connect to the WIFI 5 times at max
+    :param essid: The WIFI name to connect to, ensure it's not in double quotes but single ones
+    :param password: The WIFI password to connect, ensure it's not in double quotes but single quotes
+    :return: None
+    """
+
+    # How many attempts to connect to the WIFI
```

Comment:
Watch out for being too verbose in your comments! Try to make sure the code that you write is as readable as possible. If this is not possible, then write a comment.

If you really wanted to make it clearer, you could change the variable to `max_wifi_attempts = 5` and this clearly tells the developer reading the code what this variable is, without needing to write an additional comment. Hope you get the idea!

Reply:
yep that's a great idea, not sure why I didn't think of it